lib/elsa/producer.ex

```diff
@@ -73,10 +74,8 @@ defmodule Elsa.Producer do
 
   defp produce_sync_while_successful(registry, topic, message_chunks) do
     Enum.reduce_while(message_chunks, {:ok, 0}, fn {partition, chunk}, {:ok, messages_sent} ->
-      total_size = Enum.reduce(chunk, 0, fn {key, value}, acc -> acc + byte_size(key) + byte_size(value) end)
```

Comment:
I assume this was just taking extra time.

Reply:
yeah, it was just for a log message, which didn't seem worth it.